consensus/phases/thirdphase.go

```diff
@@ -88,9 +90,16 @@ func (tp *ThirdPhaseImpl) Execute(ctx context.Context, pulse *core.Pulse, state
 		// not needed until we implement fraud detection
 		// cells, err := packet.GetBitset().GetCells(state.UnsyncList)
 
+		handler.AddClaims(state.UnsyncList.GetClaims(ref), pulse.Entropy)
```

Comment:
move this to `for _, node := range nodes {` loop

Reply:
done